### PR TITLE
adapt query to the new model

### PR DESCRIPTION
**Pull request by lagartoverde**

### CODE REVIEW

File: app/services/regulatory-attachments-fetcher.js [hunk 1]
```diff
@@ -17,20 +17,22 @@ export default class RegulatoryAttachmentsFetcher extends Service {
   *fetch() {
     const config = getOwner(this).resolveRegistration('config:environment');
     const sparqlQuery = `
+      PREFIX gn: <http://www.geonames.org/ontology#>
```
Comment (elpoelma) [inline app/services/regulatory-attachments-fetcher.js:20]: Why is this included?

File: app/services/regulatory-attachments-fetcher.js [hunk 2]
```diff
@@ -17,20 +17,22 @@ export default class RegulatoryAttachmentsFetcher extends Service {
...
       PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
       PREFIX pav: <http://purl.org/pav/>
       PREFIX dct: <http://purl.org/dc/terms/>
       PREFIX schema: <http://schema.org/>
+      PREFIX gn: <http://data.lblod.info/vocabularies/gelinktnotuleren/>
       select distinct * where {
-        ?publishedContainer a ext:PublishedRegulatoryAttachmentContainer;
-          ext:currentVersion ?publishedAttachment.
-        ?publishedAttachment dct:title ?title.
-        ?publishedAttachment ext:content/mu:uuid ?fileId.
```
Comment (elpoelma) [inline app/services/regulatory-attachments-fetcher.js:29]: The fileId variable should still be requested right?

**Review by elpoelma — CHANGES_REQUESTED** [review]
The template titles are correctly fetched, but it is currently not possible to instantiate a new document.

**Comment by lagartoverde** [comment]
I don't know why I pushed this, as it's not finished, sorry my mind was in another place, I will set it as a draft
